src/schema/index.ts

```diff
@@ -22,13 +22,16 @@ const Schema = gql`
     user: User!
     firstName: String
     lastName: String
-    address: String!
+    address: String
+    city: String
+    country: String    
+    phoneNumber: String
+    biography: String
+    avatar: String
   }
   type Query {
     getUser(id: ID!): User
     user(username: String!): String
-  }
-  type Query {
     getAllProfiles: [Profile]
     getProfile(id: ID!): Profile
```

Comment:
@samyshyaka 
There is no need for the ID since we can get the user from the token and use to get the profile